tools/travis/run_tests.sh

```diff
@@ -34,7 +34,9 @@ if [ "$test" = "static" ]; then
 elif [ "$test" = "unit" ]; then
     dependencies="apt install -y git bzr subversion mercurial libnacl-dev libsodium-dev libffi-dev libapt-pkg-dev libarchive-dev python3-pip squashfs-tools xdelta3 && python3 -m pip install -r requirements-devel.txt -r requirements.txt codecov && apt install -y python3-coverage"
 elif [ "$test" = "integration" ] || [ "$test" = "plugins" ] || [ "$test" = "store" ] || [ "$test" = "containers" ] || [ "$test" = "snapd" ]; then
-    dependencies="apt install -y bzr curl git libnacl-dev libsodium-dev libffi-dev libapt-pkg-dev libarchive-dev mercurial python3-pip subversion squashfs-tools sudo snapd xdelta3 && python3 -m pip install -r requirements-devel.txt -r requirements.txt"
+    # snap install core fails with:
+    # - Setup snap "core" (2462) security profiles (cannot reload udev rules: exit status 2
```

Comment:
Is the core installed despite the error? The comment says it fails... but then why would we not drop the install command? I'd appreciate if you can clarify in the comment.

Reply:
I have added a few more words.

Reply:
Thanks! This is much clearer!